app/src/renderer/vuex/modules/blockchain.js

```diff
@@ -61,6 +81,66 @@ export default ({ commit, node }) => {
         })
       })
     },
+    async queryTxInfo({ state, dispatch }, height) {
+      let blockTxInfo = state.blockTxs.find(
+        b => b.length && b[0].height === height
+      )
+      if (blockTxInfo) {
+        return blockTxInfo
+      }
+      try {
+        blockTxInfo = await dispatch("getTxs", {
+          key: 0,
+          len:
+            state.block && state.block.data && state.block.data.txs
+              ? state.block.data.txs.length
+              : 0,
+          txs:
+            state.block && state.block.data && state.block.data.txs
+              ? state.block.data.txs.slice(0)
+              : []
+        })
+        blockTxInfo && state.blockTxs.push(blockTxInfo)
+        return blockTxInfo
+      } catch (error) {
+        return Promise.reject(error)
+      }
+    },
+    convertTx({ state }, txstring) {
```

Comment:
this doesn't do any manipulation on the store and should therefor maybe be in a script or someplace

Reply:
is there a clever place to put it in or just make a new file under /src/scripts?